openai/openai.go

```diff
@@ -80,19 +80,21 @@ type StreamOptions struct {
 }
 
 type ChatCompletionRequest struct {
-	Model            string          `json:"model"`
-	Messages         []Message       `json:"messages"`
-	Stream           bool            `json:"stream"`
-	StreamOptions    *StreamOptions  `json:"stream_options"`
-	MaxTokens        *int            `json:"max_tokens"`
-	Seed             *int            `json:"seed"`
-	Stop             any             `json:"stop"`
-	Temperature      *float64        `json:"temperature"`
-	FrequencyPenalty *float64        `json:"frequency_penalty"`
-	PresencePenalty  *float64        `json:"presence_penalty"`
-	TopP             *float64        `json:"top_p"`
-	ResponseFormat   *ResponseFormat `json:"response_format"`
-	Tools            []api.Tool      `json:"tools"`
+	Model               string          `json:"model"`
+	Messages            []Message       `json:"messages"`
+	Stream              bool            `json:"stream"`
+	StreamOptions       *StreamOptions  `json:"stream_options"`
+	MaxCompletionTokens *int            `json:"max_completion_tokens"`
+	MaxTokens           *int            `json:"max_tokens" deprecated:"use max_completion_tokens instead"`
```

Comment:
This is deprecated for new openai models 🙃

Reply:
`deprecated` is not struct tag I'm aware of. What you likely want is just a comment

```suggestion
	// Deprecated: Use [ChatCompletionRequest.MaxCompletionTokens]
	MaxTokens           *int            `json:"max_tokens" deprecated:"use max_completion_tokens instead"`
```